proxmox/proxmox_checks.go

```diff
@@ -63,6 +74,78 @@ func (p *ProxMox) CheckInit() error {
 
 }
 
+// CheckStorage return error when not found configured storage or any storages via ProxMox API
+func (p *ProxMox) CheckStorage(storage string) error {
+
+	var err error
+
+	var sdp sDataArr
+
+	var b bytes.Buffer
+
+	edk := errors.New("no any storages is configured")
```

Comment:
if these are used only once maybe worth moving into their respective err handling blocks so they aren't instantiated each time?

Reply:
I think I will still optimize and refactor the code about errors in proxmox handlers. I also think to make http client in one function, just substitute urls.